### PR TITLE
Fix a bug in Var

### PR DESCRIPTION
**Pull request by daemyung**
## 🙏 Describe the pull request

The backward of Var is implemented incorrectly.

## ✅ Checklist

- [x] Code follows the project's coding conventions and style.
- [x] Tests have been added or updated to cover the changes.
- [x] Documentation has been updated, if necessary.

### CODE REVIEW

File: trident/language/var.py [hunk 1]
```diff
@@ -64,19 +66,20 @@ def backward(
...
             input_ptr,
             shape=(y_size, x_size),
             strides=(y_stride, x_stride),
-            offsets=(y_offset, 0),
+            offsets=(y_offset, x_offset),
             block_shape=(1, x_block_size),
             order=(1, 0),
         )
-        input = tl.load(input_block_ptr, boundary_check=(1,), padding_option="zero")
+        input = tl.load(input_block_ptr, boundary_check=(1,))
```
Comment (mejai1206) [inline trident/language/var.py:80]: 혹시 `padding_option="zero"` 를 빼도 되는 이유를 알 수 있을까요?

Comment (daemyung) [reply trident/language/var.py:80]: 아래 `tl.where`에서 처리해줍니다.

**Review by kakao-steve-ai — APPROVED** [review]
LGTM

**Review by mejai1206 — APPROVED** [review]
LGTM

**Pull request merged**